src/Install/Console/DataProviderInterface.php

```diff
@@ -13,6 +13,8 @@
 
 interface DataProviderInterface
 {
+    public function getDebug(): bool;
```

Comment:
Can we change this to `getDebugMode()` or `isDebugMode()`, please? :smile:

Reply:
I don't think changing that interface is within the scope of this PR. Feel free to update it in another branch or create an issue.

Reply:
Will do.